### PR TITLE
yml and app.js changes

### PR DESCRIPTION
**Pull request by TomPed**
@judepereira with these changes I'm able to get the service running. What do you think?

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -73,8 +73,8 @@ http.createServer((req, res) => {
           id: data.id,
           direct: (data.event === 'responds'),
           endpoint: data.endpoint,
-          channel: data.data.channel,
-          pattern: (data.data.pattern ? new RegExp(data.data.pattern) : null),
+          channel: data.channel,
```
Comment (judepereira) [inline app.js]: @TomPed What does your request look like? It should look like the following:
{endpoint: '/', data: {channel: 'random', pattern: '/foo/'}}.

See the subscribe payload here: https://microservice.guide/schema/events/#how-it-works

File: microservice.yml [hunk 2]
```diff
@@ -104,7 +105,7 @@ commands:
             type: string
 
 environment:
-  name:
-    help: The name of the bot
-  token:
+  BOT_TOKEN:
```
Comment (judepereira) [inline microservice.yml]: Looks good :)
Created https://github.com/asyncy/platform-engine/issues/149 to implement this.

File: app.js (hunk 1 above)
Comment (TomPed) [reply app.js]: Ahh, gotcha. I'll make that change on my end then.

**Comment by TomPed** [comment]
@judepereira updated.

**Comment by judepereira** [comment]
@TomPed LGTM. Can you hold off merging this until I make the required changes in the engine?

**Comment by TomPed** [comment]
@judepereira just let me know when the engine is good.

**Comment by judepereira** [comment]
@TomPed Yes, probably this week. I need to change it for all services as well.

**Comment by judepereira** [comment]
@TomPed Done. Can I delete this branch?

**Pull request merged**